zerver/webhooks/newrelic/view.py

```diff
@@ -1,169 +1,264 @@
 # Webhooks for external integrations.
+
+
+from django.core.exceptions import ValidationError
 from django.http import HttpRequest, HttpResponse
-from django.utils.translation import gettext as _
 
 from zerver.decorator import webhook_view
-from zerver.lib.exceptions import JsonableError
+from zerver.lib.exceptions import UnsupportedWebhookEventTypeError
 from zerver.lib.response import json_success
 from zerver.lib.typed_endpoint import JsonBodyPayload, typed_endpoint
 from zerver.lib.validator import (
     WildValue,
+    check_float,
     check_int,
     check_list,
     check_none_or,
     check_string,
-    check_string_fixed_length,
+    check_string_in,
     check_union,
 )
 from zerver.lib.webhooks.common import check_send_webhook_message, unix_milliseconds_to_timestamp
 from zerver.models import UserProfile
 
-# Newrelic planned to upgrade Alert Notification Channels to Workflows and Destinations
-# https://discuss.newrelic.com/t/plan-to-upgrade-alert-notification-channels-to-workflows-and-destinations/188205
-# This view will handle both old and new format but will keep it easy to delete the old code
-# once it is EOLed by the end of June, 2023
+PRIORITIES = {
+    "CRITICAL": ":red_circle:",
+    "HIGH": ":orange_circle:",
+    "MEDIUM": ":yellow:",
+    "LOW": ":blue_circle:",
+}
+
+TIME_STAMPS_MAP: dict[str, str] = {
+    "CREATED": "createdAt",
+    "ACTIVATED": "activatedAt",
```

Comment:
Correct me if I'm wrong, but this "ACTIVATED" key doesn't seem to be used at all in the parsing of the payload or in getting the status.

Reply:
hmm, yeah it is one of the fields from the recommended base payload template. But it looks like we can leave it out